indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/wallet/IndySDKWalletUser.kt

```diff
@@ -0,0 +1,416 @@
+package com.luxoft.blockchainlab.hyperledger.indy.wallet
+
+import com.luxoft.blockchainlab.hyperledger.indy.helpers.TailsHelper
+import com.luxoft.blockchainlab.hyperledger.indy.models.*
+import com.luxoft.blockchainlab.hyperledger.indy.utils.SerializationUtils
+import com.luxoft.blockchainlab.hyperledger.indy.utils.getRootCause
+import org.hyperledger.indy.sdk.anoncreds.Anoncreds
+import org.hyperledger.indy.sdk.anoncreds.DuplicateMasterSecretNameException
+import org.hyperledger.indy.sdk.did.Did
+import org.hyperledger.indy.sdk.ledger.Ledger
+import org.hyperledger.indy.sdk.pairwise.Pairwise
+import org.hyperledger.indy.sdk.wallet.Wallet
+import org.slf4j.LoggerFactory
+import java.util.concurrent.ExecutionException
+
+/**
+ * This is an implementation of [WalletUser] which uses indy sdk standard [org.hyperledger.indy.sdk.wallet.Wallet]
+ *  and [org.hyperledger.indy.sdk.anoncreds.Anoncreds] API
+ *
+ * @param wallet [Wallet] - user's wallet
+ * @param didConfig [DidConfig] - what did we should use to perform operations
+ * @param tailsPath [String] - path to the directory with tails files (they will be generated when revocation-stuff is done)
+ */
+class IndySDKWalletUser(
+    val wallet: Wallet,
+    didConfig: DidConfig = DidConfig(),
+    val tailsPath: String = "tails"
+) : WalletUser {
+
+    val did: String
+    val verkey: String
```

Comment:
Lets move last 2 fileds to the interface?

Reply:
Done.

Reply:
Also created this ticket https://luxproject.luxoft.com/jira/browse/CTO-186